docs/releases/2.1.txt

```diff
@@ -169,6 +169,9 @@ Models
 * A ``BinaryField`` may now be set to ``editable=True`` if you wish to include
   it in model forms.
 
+* The new database function :class:`~django.db.models.functions.NullIf`
+  provides the NULLIF function.
```

Comment:
```The new :class:`~django.db.models.functions.NullIf` database function returns `None` if the two provided expressions are equal.```

Reply:
I think it should be double ticks around `None`, but otherwise I copied it in. Thanks.